metastore_db/README_DO_NOT_TOUCH_FILES.txt

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
You have a lot of extra files in `./metastore_db/` and `./spark-warehouse/` that I assume were generated by your script.  They didn't generate when I ran the code on a Linux distribution, so maybe they're platform dependent?  Either way, you may want to add these folders to `.gitignore`.